.github/fabricbot.json

```diff
@@ -1,535 +0,0 @@
-{
-  "version": "1.0",
-  "tasks": [
-    {
-      "taskType": "trigger",
-      "capabilityId": "InPrLabel",
-      "subCapability": "InPrLabel",
-      "version": "1.0",
-      "config": {
-        "label_inPr": "in-pr",
-        "taskName": "Add in-pr label to issues"
-      }
-    },
-    {
-      "taskType": "scheduled",
```

Comment:
@jeffhandley This scheduled task seems to have been lost in the transition to [GitHub Policy Service](https://eng.ms/docs/more/github-inside-microsoft/policies). Do you know if it's possible to have PRs from main to live created automatically on a schedule with GPS? FYI @BillWagner

Reply:
Nevermind, I see it's working in the dotnet-api-docs repo, so I'll copy that config here.